sktm/db.py

```diff
@@ -40,15 +40,20 @@ def createdb(self, db):
         c.executescript("""
                 PRAGMA foreign_keys = on;
 
+                /* Base Git repositories */
                 CREATE TABLE baserepo(
                   id INTEGER PRIMARY KEY,
                   url TEXT UNIQUE
                 );
 
+                /* Patch sources (Patchwork projects) */
                 CREATE TABLE patchsource(
                   id INTEGER PRIMARY KEY,
+                  /* Patchwork base URL */
                   baseurl TEXT,
+                  /* Numeric Patchwork project ID */
                   project_id INTEGER,
+                  /* TODO Document or remove */
```

Comment:
Checking git history, this seems to be a residue of adding events API support and neither `date` nor the `set_event_date` and `get_last_event_date` methods which use it aren't used anymore. Now the `nsince` attribute is used to save these information. I'm fine with removing the column and methods that use it.

Reply:
OK, cool :) I'll change this to just `TODO: Remove`.